crates/bin/pclientd/src/lib.rs

```diff
@@ -338,6 +338,8 @@ impl Opt {
                     .add_service(tonic_web::enable(stake_query_proxy))
                     .add_service(tonic_web::enable(compact_block_query_proxy))
                     .add_service(tonic_web::enable(tendermint_proxy_proxy))
+                    // TODO: should we add the IBC services here as well? they will appear
+                    // in reflection but not be available.
```

Comment:
As a general rule, I'd say any services that _can_ be exposed in both `pd` and `pclientd`, meaning they'll work equally well in both contexts, probably should be. Otherwise we're starting down a path where there are odd discrepancies that are hard to document. Not worth blocking merge here though, and I appreciate you adding this TODO.